src/social-web/routes/feed/inspector.tsx

```diff
@@ -26,16 +27,21 @@ const RenderHTML = ( { html }: { html: string } ) => {
 	return <div dangerouslySetInnerHTML={ { __html: decoded } } />;
 };
 
-export default function FeedInspector( { id, onClose }: FeedInspectorProps ) {
+export default function FeedInspector( { id, onClose, onTagClick }: FeedInspectorProps ) {
 	const { defaultAvatar } = useSettings();
 	const { record: post, isResolving: isLoading } = useEntityRecord< FeedPost >( 'postType', 'ap_post', id );
 	const { records: comments, isResolving: isLoadingComments } = useEntityRecords< Comment >( 'root', 'comment', {
 		post: id,
-		per_page: 100,
 		order: 'asc',
 		orderby: 'date',
 	} );
 
+	// Fetch tag terms if the post has tags
+	const tagIds = post?.ap_tag || [];
+	const { records: tags } = useEntityRecords( 'taxonomy', 'ap_tag', {
```

Comment:
```suggestion
	const { records: Term[] } = useEntityRecords( 'taxonomy', 'ap_tag', {
```
Do we have `tag` defined anywhere? If not we could probably reuse `@wordpress/core-data`'s `Term` type